src/standalone/json-parse/main.cpp

```diff
@@ -0,0 +1,59 @@
+#include <nlohmann/json.hpp>
```

Comment:
I think this example should not be placed inside the repo main branch.

Reply:
Ok, I will exlude it